### DISCUSSION TITLE
Why zagjs/pressable doesn't work in newer version, why is the documentation gone? 😢

### DISCUSSION THREAD

**Discussion by thmsfchtnr** [chakra-ui/zag#1487] [Q&A]
Hi everyone! 
I've used zagjs/pressable in the past to implement a button state machine similar to React Spectrum's usePress, but I can't find it in the newer versions of ZAG.js. 
Has it been deprecated or moved? If so, what's the recommended alternative now? 
Thanks for your help!

**Comment by segunadebayo** [comment]
Hey @thmsfchtnr,

The pressable machine was deprecated. We couldn't justify the complexity it brought vs the benefit it gives; it was also quite buggy across browsers, forcing us to handle the nuances in the machine. tldr: it wasn't worth the effort.

I recommend using react-spectrum's usePress for this, for now. We might look into implementing React Native's logic for pressable in the future.

**Reply by thmsfchtnr** [reply]
Hey thank you for the quick reply that's really unfortunate. Iam using vuejs is there an alternative I can use or do I have to build it myself 😅
